debian/Dockerfile

```diff
@@ -73,19 +74,18 @@ ADD --chmod=0755 https://github.com/mlocati/docker-php-extension-installer/relea
 
 # Install Required PHP extensions.
 RUN install-php-extensions \
+#RUN docker-php-ext-install \
     bcmath \
-    # curl \ Already installed
     exif \
     gd \
     gmp \
     imagick \
-    # mbstring \ Already installed
     mysqli \     
     opcache \
     pcntl \
     pdo_mysql \
     redis \
-    # saxon \ outdated
+    saxon-${saxon} \
```

Comment:
Does this really work?

I was under the impression that you would still need libsaxon in the system also? Or does this extension also pull this in?

I'd be pretty happy if it did!